### ISSUE TITLE
Weird behavior with Alone in the Dark: The New Nightmare

### ISSUE THREAD

**Issue by instinctualjealousy** [samuelgr/Xidi#70]
I'm able to get Xidi to hook and log, and I have a valid Xidi.ini I made that gets no complaint prompt. It is worth noting I use a manifest workaround to convince games to load DLLs when they refuse to, this game is one of those games that would need Hookshot otherwise.

From what I can tell, the mapping changes I've applied within Xidi are not getting used (triggers won't assign as if they're axes still, keyboard mappings don't function/real keyboard does)- the log is very dead as well, as seen here:
[Xidi_DInput8_alone4.exe_14092.log](https://github.com/samuelgr/Xidi/files/12850223/Xidi_DInput8_alone4.exe_14092.log)

This is after going to the controller configuration menu, configuring controls and the like, and closing the game. Looking at the alone4.exe process, it does weird stuff like load dinput.dll and dinput8.dll (I'm hooking with 8), I can confirm leading the game with dinput.dll as an entrypoint doesn't change anything. I can also confirm another dinput8 wrapper I have does hook and achieve its purpose with this game (with the same manifest workaround to get it to load, of course).
```
[Mapper]
Type		= AITDTNT

[CustomMapper:AITDTNT]
Template	= StandardGamepad

DpadUp		= Keyboard(Up)
DpadDown	= Keyboard(Down)
DpadLeft	= Keyboard(Left)
DpadRight	= Keyboard(Right)

ButtonBack	= Keyboard(Escape)
ButtonStart	= Keyboard(Enter)

[Log]
Enabled		= yes
Level		= 4

[Workarounds]
ActiveVirtualControllerMask = 0x01
```
I've even tried the ActiveVirtualControllerMask as seen here, in case it was that. I do seem to remember the game cared about which controller was set to default when used wrapperless?

**Comment by samuelgr** [comment]
Have you tried the winmm form of Xidi? If memory serves, some games use DirectInput for keyboard/mouse but winmm for joysticks and gamepads.

**Comment by samuelgr** [comment]
I confirmed for myself that the GOG version of this game uses winmm to access the game controller. Putting the winmm.dll form of Xidi into the same directory as the executable is enough to get Xidi loaded and working (absent any special manifest magic, though it shouldn't be hard to make it work with Xidi's winmm form too).

The game gets the joystick position using the older [`joyGetPos`](https://learn.microsoft.com/en-us/windows/win32/api/joystickapi/nf-joystickapi-joygetpos) function, which technically only supports 4 buttons. However, I found with 4659936bec8af9530a71a980b9602cf16710f2f2 that the game will nevertheless actually respond to 8 buttons.

I tried your config file, too, and it seems to work fine via winmm.
